examples/README.md

```diff
@@ -0,0 +1,544 @@
+### Tutorial 
+This tutorial shows how to create, build, install and test  chaincode using the Fabric-Private-chaincode(FPC) framework.  This assumes familiarity with the [concepts](https://hyperledger-fabric.readthedocs.io/en/release-1.4/whatis.html#) and [the programming model](https://hyperledger-fabric.readthedocs.io/en/release-1.4/chaincode.html) in Hyperledger Fabric v1.4.
+
+
+Refer to [package shim](https://godoc.org/github.com/hyperledger/fabric/core/chaincode/shim) for the GoDoc for shim interface provided by Fabric.  The FPC programming model for chaincode provides a simpler version of the _shim_ SDK provided by Hyperledger Fabric in Go and node.js.  FPC provides a C++ interface  to access its state variables and transaction context through [shim.h](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/ecc_enclave/enclave/shim.h).  The standard commands are similar to ones in Fabric.  To ensure confidentiality of the arguments passed to the chaincode, the arguments are encrypted while using FPC SDK.    
+
+This example illustrates a simple usecase where the chaincode is used to store a single asset, `asset1` in the ledger and then retrieve the latest value of `asset1`.  Here are the steps to accomplish this: 
+* Develop chaincode
+* Launch Fabric network
+* Setup _Enclave Registry_ 
+* Install and instantiate chaincode on the peer
+* Invoke transactions (`storeAsset` and `retrieveAsset`)
+* Shut down the network
+
+Please refer to [Architecture and Components](https://github.com/hyperledger-labs/fabric-private-chaincode#architecture-and-components) for more details on Enclave Registry.
+
+### Prequisites
+This tutorial presumes that the repository in https://github.com/hyperledger-labs/fabric-private-chaincode has been installed as per [README.md](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/README.md) in FPC-INSTALL-DIR which is in the $GOPATH folder. 
+
+### Develop chaincode
+* Create a folder named `helloworld`  in FPC-INSTALL-DIR/examples.
+```
+cd $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode/examples
+mkdir helloworld
```

Comment:
if we suggest to create the example inside the repo, maybe we want to add `examples/helloworld` to `.gitignore?